packages/tables/resources/views/index.blade.php

```diff
@@ -709,6 +709,7 @@ class="filament-table-header-cell-{{ \Illuminate\Support\Str::of($column->getNam
                                 $recordAction = $getRecordAction($record);
                                 $recordKey = $getRecordKey($record);
                                 $recordUrl = $getRecordUrl($record);
+                                $recordBulkActionEnabled = $getRecordBulkActionEnabled($record);
```

Comment:
This should follow the convention and be `$isRecordBulkSelectable()` or similar.
Also it should only be evaluated if `$isSelectionEnabled` is `true`.

Reply:
Changed. The $isSelectionEnabled is evaluated first thing in the method.

Reply:
Yes, but we already have `$isSelectionEnabled`. No need to run the code again.